app/main.py

```diff
@@ -1,8 +1,85 @@
 class Car:
-    # write your code here
-    pass
+    max_comfort_class = 7
+    min_comfort_class = 1
+    max_clean_mark = 10
+    min_clean_mark = 1
+
+    def __init__(self, comfort_class, clean_mark, brand):
+        if comfort_class > self.max_comfort_class \
+                or comfort_class < self.min_comfort_class:
+            raise ValueError(
+                f'comfort_class value is {comfort_class},'
```

Comment:
We use double quotes by default in this course. So it would be better to use them in error messages. 
Actually, single quotes are usually fine but It is important to stick to one style